cmd/beach/cmd/resource-download.go

```diff
@@ -113,7 +113,7 @@ func handleResourceDownloadRun(cmd *cobra.Command, args []string) {
 			log.Error(err)
 		} else {
 			source := bucket.Object(attributes.Name)
-			targetPath := filepath.Dir(filepath.Join(resourcesPath, getRelativePersistentResourcePathByHash(attributes.Name)))
+			targetPath := filepath.Join(resourcesPath, getRelativePersistentResourcePathByHash(attributes.Name))
```

Comment:
Would it make sense to leave `Dir` in place but join the filename "correctly", i.e. by adding a `/ ` in between? 🤔

Reply:
I thought the same, but filepath.Join removes it.

Reply:
https://pkg.go.dev/path/filepath#Join

Reply:
Or do you mean 
```
targetPath := filepath.Dir(filepath.Join(resourcesPath, getRelativePersistentResourcePathByHash(attributes.Name), attributes.Name))
```

Reply:
Sorry for the delay, I had to look at the code in full. I got the issue now and have it fixed…